_includes/toc.html

```diff
@@ -248,6 +248,7 @@
         <a data-toggle="collapse" class="menu collapsed" href="#collapse_quantiles_studies">Quantiles Studies</a>
       </p>
       <div class="collapse" id="collapse_quantiles_studies">
+        <li><a href="{{site.docs_dir}}/QuantilesStudies/KllSketchVsTDigest.html">•KLL sketch vs t-digest</a></li>
```

Comment:
why this was restored? outdated clone, perhaps?

Reply:
It [still exists](https://github.com/apache/datasketches-website/blob/master/src/main/resources/docgen/toc.json#L212) in master branch `toc.json`.
I thought it was inadvertently removed and left in the PR, but please let me know if it was supposed to be removed.

Reply:
I seems like we forgot to remove it from json, but it is not visible on the web site, which is intentional.

Reply:
Sounds good, removed from 'toc.json' and the html file.